central/reports/scheduler/v2/reportgenerator/report_gen_impl.go

```diff
@@ -107,7 +110,14 @@ func (rg *reportGeneratorImpl) generateReportAndNotify(req *ReportRequest) error
 	}
 
 	// Format results into CSV
-	zippedCSVData, empty, err := common.Format(deployedImgData, watchedImgData)
+	reportConfig, found, err := rg.configDatastore.GetReportConfiguration(reportGenCtx, req.ReportSnapshot.GetReportConfigurationId())
+	if !found {
+		return errors.Wrapf(errox.NotFound, "Report configuration id %s not found", req.ReportSnapshot.GetReportConfigurationId())
+	}
+	if err != nil {
+		return err
+	}
+	zippedCSVData, empty, err := common.Format(deployedImgData, watchedImgData, reportConfig.GetName())
```

Comment:
No need to get the report configuration from datastore. The config name is present in the `req.ReportSnapshot.Name`